Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1599,7 +1599,7 @@ ExceptionOr<void> CanvasRenderingContext2DBase::drawImage(WebCodecsVideoFrame& f
 
     auto normalizedDstRect = normalizeRect(dstRect);
     bool repaintEntireCanvas = rectContainsCanvas(normalizedDstRect);
-    didDraw(repaintEntireCanvas, normalizedDstRect);
+    didDraw(repaintEntireCanvas, normalizedDstRect, defaultDidDrawOptionsWithoutPostProcessing());
```

Comment:
so if the video element captures another canvas, then this draw will remove the postprocessing

Reply:
Good catch for this change. I added a FIXME in case we can identity the same sources (e.g., ImageBitmap, HTMLVideoElement).